### PR TITLE
Create Template_Technical_Charter_CHIPS_Workgroups.md

### PR DESCRIPTION
**Pull request by mcockrell-google**
@mgielda @robmains  Adding a Markdown version of the [proposal](https://docs.google.com/document/d/186wqh9kjToZkrDb9LizErzQNumhliwJl7ZrGX06u1Ts/edit?usp=sharing).

### CODE REVIEW

**Comment by mgielda** [comment]
Several votes in, but we have some feedback from Dave Kehlet we are discussing. I will suggest a minor, non controversial change while we discuss any bigger ones.

File: Template_Technical_Charter_CHIPS_Workgroups.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Sharing any necessary meeting minutes in a public GitHub repository, a public Wiki, or on a public website for the Workgroup within a reasonable period of time after the meeting;
+- Responding to technical questions on GitHub; 
+- Setting the technical criteria for use of the Workgroup marks (if applicable);
+- Determining strategy and promoting the adoption of the Workgroup technology through documentation, talks, or other marketing;
+- Determining whether additional public, open source repositories should be created for the Workgroup; and
+- Determining whether any Workgroup open source collaborations should be wound down and archived on GitHub.
+
+## Eligibility
+
+Companies who have joined the CHIPS Alliance and signed the CHIPS Alliance Participation Agreement and CHIPS Alliance Contributor License Agreement are eligible to nominate developer representatives to join the Workgroup TAC.
```
Comment (mgielda) [inline Template_Technical_Charter_CHIPS_Workgroups.md:42]:
```suggestion
Organizations who have joined the CHIPS Alliance and signed the CHIPS Alliance Participation Agreement and CHIPS Alliance Contributor License Agreement are eligible to nominate developer representatives to join the Workgroup TAC. 
```

**Comment by mcockrell-google** [comment]
@mgielda - how are we looking for merging the PR?

**Comment by mcockrell-google** [comment]
andreslagarcavilla - for visibility

**Comment by mgielda** [comment]
Added some changes after the TSC's feedback. Restarting the vote now.

**Review by hzeller — APPROVED** [review]

**Pull request merged**